src/test/java/tests/FilterEvalTests.java

```diff
@@ -824,6 +825,40 @@ public void filterLabelMatching() {
         assertFalse(Qualifier.labelMatches(".", "p.high"));
     }
 
+    @Test
+    public void QualifierProcess_useInvalidUsername_getUserWarning() {
```

Comment:
Typically the test will be in a `Test[ClassName]` class (i.e. each class should have a corresponding `Test*` class), so the test method need not mention the class name.